app/services/find_sync/sync_courses_from_find.rb

```diff
@@ -96,9 +96,6 @@ def assign_course_attributes_from_find(course, find_course)
       course.exposed_in_find = find_course.findable?
       course.subject_codes = find_course.subject_codes
       course.funding_type = find_course.funding_type
-      # these two fields are in the API docs, but not actually in the API yet
-      course.program_type = find_course.try(:program_type)
```

Comment:
I added program type to the v3 a while ago as Mike needed it for analysis.

https://github.com/DFE-Digital/teacher-training-api/blob/master/app/serializers/api/v3/serializable_course.rb#L24

Reply:
I'll reinstate the `program_type` sync; it doesn't seem to be the cause of any extra audit entries. I was originally concerned that it might conflict with https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/services/teacher_training_public_api/sync_courses.rb#L33

Reply:
Yea I'd missed that. Do we actually need the quals? If so we can just add it to the public API. We've gone for this far without is so i'd assume not?